### PR TITLE
YShyptur - Task 1

### PR DESCRIPTION
**Pull request by YevhenShyptur**

### CODE REVIEW

File: Bootcamp/Lecture1/Homework/shyptur/myfirstproject/Program.cs [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+namespace myfirstproject
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            task10();
+
+        }
+        static void task2()
```
Comment (olehkrut) [inline Bootcamp/Lecture1/Homework/shyptur/myfirstproject/Program.cs:12]: method name should be written in pascal case.

File: Bootcamp/Lecture1/Homework/shyptur/myfirstproject/Program.cs [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+            int maxvalue = numbers1;
+            int minvalue = numbers1;
+
+            if (numbers2 > maxvalue)
+            {
+                maxvalue = numbers2;
+            }
+            else
+            {
+                if (numbers3 > maxvalue)
```
Comment (olehkrut) [inline Bootcamp/Lecture1/Homework/shyptur/myfirstproject/Program.cs:61]: avoid nesting clauses, you can use `else if (number3 > maxvalue)`

File: Bootcamp/Lecture1/Homework/shyptur/myfirstproject/Program.cs [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+            }
+            else
+            {
+                Console.WriteLine(false);
+            }
+        }
+        static void task8()
+        {
+            string str = Convert.ToString(Console.ReadLine());
+            if (str.Substring(0, 2) == "C#")
```
Comment (olehkrut) [inline Bootcamp/Lecture1/Homework/shyptur/myfirstproject/Program.cs:120]: there is a method called `StartsWith`

File: Bootcamp/Lecture1/Homework/shyptur/myfirstproject/Program.cs [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+        static void task10()
+        {
+            double[] numbers = new double[10];
+            double sum = 0;
+            for (int i = 0; i < 10; i++)
+            {
+                numbers[i] = Convert.ToDouble(Console.ReadLine());
+                sum += numbers[i];
+            }
+            Console.WriteLine("Sum=" + sum + "    avg=" + sum / 10);
```
Comment (olehkrut) [inline Bootcamp/Lecture1/Homework/shyptur/myfirstproject/Program.cs:144]: you can use string interpolation $"Sum={sum}, avg={sum / 10}"

**Review by olehkrut — COMMENTED** [review]
Looks good! 
Several minor suggestions: 

-  naming conventions: method names, namespace names
- formatting suggestion: do not spam with new lines ) 
- formatting suggestion: use new lines to separate each method

File: Bootcamp/Lecture1/Homework/shyptur/myfirstproject/Program.cs [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+        static void task2()
+        {
+            int number1 = Convert.ToInt32(Console.ReadLine());
+            int number2 = Convert.ToInt32(Console.ReadLine());
+            int number3 = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine(number1 * number2 * number3);
+        }
+        static void task3()
+        {
+            string str = Convert.ToString(Console.ReadLine());
```
Comment (m96roman) [inline Bootcamp/Lecture1/Homework/shyptur/myfirstproject/Program.cs:21]: Console.ReadLine() returns string. No need to do convert.

File: Bootcamp/Lecture1/Homework/shyptur/myfirstproject/Program.cs [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+        }
+        static void task9()
+        {
+            double radius = Convert.ToDouble(Console.ReadLine());
+            Console.WriteLine("perimetr=" + Math.PI * 2 * radius);
+            Console.WriteLine("area=" + Math.PI * radius * radius);
+        }
+        static void task10()
+        {
+            double[] numbers = new double[10];
```
Comment (m96roman) [inline Bootcamp/Lecture1/Homework/shyptur/myfirstproject/Program.cs:137]: You don't need array here. 
You can just do sum += ConvertToDouble(Console.ReadLine());

File: Bootcamp/Lecture1/Homework/shyptur/myfirstproject/Program.cs (hunk 1 above)
Comment (YevhenShyptur) [reply Bootcamp/Lecture1/Homework/shyptur/myfirstproject/Program.cs]: fixed

**Review by olehkrut — APPROVED** [review]

**Pull request merged**